modules/ROOT/pages/bolt/message.adoc

```diff
@@ -716,6 +716,33 @@ Clients receiving a `FAILURE` in response to `LOGOFF` should treat that connecti
 FAILURE {'code': 'Neo.ClientError.Request.Invalid', 'message': "Message 'LogoffMessage{}' cannot be handled by a session in the FAILED state."}
 ----
 
+[[messages-telemetry]]
+=== Request message `TELEMETRY`
+
+The `TELEMETRY` message contains an int which represents an api type that has been called on the driver, this is then
```

Comment:
Maybe note in here that TELEMETRY is only sent when enabled/processed on both server and driver